docs/introduction/getting_started.rst

```diff
@@ -67,18 +106,25 @@ If you need to create an uncoupled spin system, simply omit the ``couplings`` at
 
     # Create the SpinSystem object
     spin_system = SpinSystem(
-        sites=[H_site, C_site],
-        couplings=[coupling],
+        sites = my_sites,
+        couplings = [coupling],
     )
 
-Thats it! We have created a spin system whose spectrum is ready to be simulated.
+That's it! You have created a spin system whose spectrum is ready to be simulated.
+If you had wanted to create an uncoupled spin system, simply omit the
+``couplings`` attribute.
+
 
-Methods
--------
+Method
+------
 
-A :ref`method_documentation` object describes an NMR method. For this introduction, we will use
-the :py:class:`~mrsimulator.method.lib.BlochDecaySpectrum` which is one of the pre-defined methods.
-Some attributes of the method still need to be provided as seen below.
+A :ref:`method_documentation` object in ``mrsimulator`` describes an NMR method.
```

Comment:
Also a convention we should decide on is wether to write simply mrsimulator (in standard text) or ``mrsimulator`` (code text) when referencing the package. @pjgrandinetti @deepanshs thoughts?

Reply:
i agree.  i don't think code text is right.